src/main/resources/application.yml

```diff
@@ -2,9 +2,10 @@ spring:
   application:
     name: hmpps-assessments-api
   cache:
-    cache-names: users
-    caffeine:
-      spec: expireAfterWrite=2m
+    type: redis
+    cache-names: verifyUserAccess, offender, roshRiskSummary, riskRegistrations
+    redis:
+      time-to-live: 300000
```

Comment:
Is this in milliseconds?

Reply:
It is. I can't see where that is stated in the docs, but here you can see 600000 is 10 minutes https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-caching.html#boot-features-caching-provider-redis